mkosi/resources/mkosi.md

```diff
@@ -1021,6 +1021,20 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 : Even if no EFI bootloader is installed, we still need an ESP for BIOS
   boot as that's where we store the kernel, initrd and grub modules.
 
+`ShimBootloader=`, `--shim-bootloader=`
+
+: Takes one of `none`, `unsigned` or `signed`. Defaults to `none`. If
+  set to `none`, shim will not be installed. If set to `unsigned`, mkosi
```

Comment:
Oxford comma please.

I think that "installed" should be clarified here. Because there's package installation and the installation from the installed package to the ESP.